src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```
@@ -0,0 +1,28 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CategoryServiceImpl implements CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    @Override
+    public Category findById(Long id) {
+        return categoryRepository.findById(id).get();
```

let's throw an exception in case category isn't found